roadmap.js

```diff
@@ -0,0 +1,361 @@
+              if (support !== true) throw new TypeError();
+              text = icon('yes');
+              cls = STYLE_MAP['no'];
+              // Keep in sync with `buildTooltip`
+              if (!note) note = '✓ Supported, introduced in unknown version';
+            }
+
+            // We need an extra div because <td> can't be `display: flex`
+            const box = h('div', { className: `feature-cell ${cls}` }, [text]);
+            const cell =  h('td', {
+              headers: [idMap['table-col'](browserName), idMap['table-row'](featName)].join(' ')
+            }, [box]);
+
+            if (note && note2index.has(note)) {
+              cell.tabIndex = 0;  // focusable
+              const index = note2index.get(note);
+              const [noteId, refLink] = createNoteRef(index);
+              box.appendChild(h('sup', {}, [refLink]));
+
+              const noteItem = document.getElementById(noteId);
+              if (noteItem) {
+                cell.addEventListener('mouseenter', () => noteItem.classList.add('ref-highlight'));
+                cell.addEventListener('mouseleave', () => noteItem.classList.remove('ref-highlight'));
+              }
+            }
+
+            const descId = `table-cell-desc-${counter++}`;
+            tooltipTasks.push((createPopper) => {
+              cell.appendChild(buildTooltip(createPopper, cell, browserName, note, descId));
+              cell.setAttribute('aria-describedby', descId);
+            });
+
+            return cell;
+          })
+        ])
+      );
+      tBody.lastElementChild.setAttribute('aria-describedby', idMap['table-row'](featName));
+    }
+  }
+
+  function buildTooltip(createPopper, cell, browserName, note, descId) {
+    const title = h('h4', { className: 'feature-tooltip-title' }, [browserName]);
+    title.setAttribute('aria-hidden', 'true');
+
+    const noteTag = h('div', { className: 'feature-tooltip-desc', id: descId });
+    noteTag.innerHTML = renderNote(note);
+
+    if (note.includes('introduced in unknown version')) {
+      noteTag.append(' ', h(
+        'a',
+        { href: 'https://github.com/WebAssembly/website/blob/master/features.json', target: '_blank' },
+        ['(contribute data)']
+      ));
+    }
+
+    const arrow = h('div', { className: 'feature-tooltip-arrow' });
+    const tooltip = h('div', { className: 'feature-tooltip', role: 'tooltip' }, [
+      title,
+      noteTag,
+      arrow
+    ]);
+
+    const popper = createPopper(cell, tooltip, {
+      placement: 'right',
+      modifiers: [
+        {
+          name: 'offset',
+          options: {
+            offset: [0, 6],
+          },
+        },
+        {
+          name: 'arrow',
+          options: {
+            element: arrow,
+            padding: 2,
+          },
+        },
+        {
+          name: 'eventListeners',
+          enabled: false
+        },
+      ]
+    });
+
+    function show() {
+      tooltip.setAttribute('data-show', '');
```

Comment:
Is data attribute used because Popper requires it? Or can we just use a class here?